src/constants/footer.js

```diff
@@ -0,0 +1,30 @@
+import insta from '../images/instagram.svg';
+
+export const FooterData = {
+    blocks: [
+        {
+            title: "Let's get in touch",
+            text1: 'Hello@AwesomeContainers.com',
+            text2: 'Address: 71 UBI Road 1, #08-42, \nOxley Bizhub, Singapore 408732',
+            size: 'small',
+            margin: 'bottom'
+
+        },
+        {
+            title: 'Follow us',
+            img1: insta,
+            text: 'AwesomeContainers',
+            link: null
+            // need instagram link for this element
+        },
+        {
+            title: 'Where we work',
+            text1: 'We currently operate in: \nSingapore',
+            text2: 'Planning to launch in 2022: \nUK, EU, OSEAN, China, Vietnam, Malaysia, \nTaiwan'
+        }
+    ],
+    links: [
+        'Awesome Container Company © 2022',
```

Comment:
Is this ever used? I notice the HTML still has this as well, but company is misspelled in the HTML